tools/hpcc_aks_util.pl

```diff
@@ -0,0 +1,772 @@
+CPUs Allocated  - Total CPU cores allocatable to pods (slightly less than capacity due to system overhead)
+Alloc(%)        - Percentage of total capacity that is allocatable (Allocated/Capacity * 100)
+CPUs In Use     - Actual CPU cores currently being consumed by all workloads
+InUse(%)        - Percentage of allocated CPU being used (In Use/Allocated * 100)
+HPCC(%)         - Percentage of allocated CPU requested by HPCC components
+Mem Alloc(Mi)   - Total memory allocated/available across all nodes in the pool (in MiB)
+Mem Used(Mi)    - Total memory currently being used across all nodes in the pool (in MiB)
+Mem(%)          - Percentage of memory currently being used across all nodes in the pool
+Cost/Hr         - Estimated hourly cost for all nodes in this pool (USD, with applied discounts)
+
+POD DETAILS (when --show-pods is used):
+======================================
+
+Pod Name        - Kubernetes pod name
+Node            - Specific node where the pod is running
+Phase           - Pod lifecycle phase (Running, Pending, Failed, etc.)
+Status          - Pod readiness status (Ready/NotReady)
+CPU Req         - CPU cores requested by this pod (affects scheduling)
+Mem Req(Mi)     - Memory requested by this pod in MiB (affects scheduling)
+
+NODE DETAILS (when --show-nodes is used):
+========================================
+
+Node Name       - Kubernetes node name within the pool
+CPU Capacity    - Total CPU cores available on this specific node
+CPU Allocatable - CPU cores available for pod scheduling (after system overhead)
+CPU Used        - Actual CPU cores currently being consumed on this node
+CPU(%)          - Percentage of allocatable CPU being used on this node
+Mem Capacity(Mi)- Total memory available on this specific node (in MiB)
+Mem Alloc(Mi)   - Memory available for pod scheduling (after system overhead, in MiB)
+Mem Used(Mi)    - Actual memory currently being consumed on this node (in MiB)
+Mem(%)          - Percentage of allocatable memory being used on this node
+VM Size         - Azure VM instance type for this node
+
+For each node, ALL pods running on that node are listed (from all namespaces), showing:
+Namespace       - Kubernetes namespace the pod belongs to
+Pod Name        - Kubernetes pod name
+Phase           - Pod lifecycle phase (Running, Pending, Failed, etc.)
+Status          - Pod readiness status (Ready/NotReady)
+CPU Req         - CPU cores requested by this pod (affects scheduling)
+Mem Req(Mi)     - Memory requested by this pod in MiB (affects scheduling)
+
+COST CALCULATIONS:
+=================
+
+The script uses East US 2024 pricing as baseline and applies configurable discounts:
+
+--ea-discount           Applied to most VM series (D-series v5, v4 smaller, E-series, F-series, B-series)
+--ds_v4_family_discount Applied specifically to D16ds v4 and larger instances
+
+Default discounts are 10% each. Costs are estimates and actual pricing may vary.
```

Comment:
would it better to not default to 10% discount like this. Default to published costs, unless --discount's apply?